Gemfile

```diff
@@ -11,8 +11,6 @@ gem "pg"
 gem "puma", "~> 4.3"
 # Use SCSS for stylesheets
 gem "sass-rails", "~> 5"
-# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
-gem "webpacker", "~> 4.0"
```

Comment:
Do we need to add anything back in to make sure the assets are being compiled? Or is this enough?

Also we don't have CI set up for this, which is ugh... I'll pull it and make sure it works before approving.